core/perf_test/CMakeLists.txt

```diff
@@ -49,11 +49,19 @@ SET(SOURCES
   )
 
 IF(Kokkos_ENABLE_HIP)
-# FIXME requires TeamPolicy
+# FIXME HIP requires TeamPolicy
```

Comment:
We use the FIXME_HIP token

Reply:
Apparently you don't because it only said FIXME before ;-)